### PR TITLE
Issue#4: Updated the getTournamentCodes to be less confusing

### PR DESCRIPTION
**Pull request by CoreyTrombley**
This is a minor update to the tournaments to reduce confusion.

This pull request is for issue#4.

### CODE REVIEW

**Comment by lucalaissue** [comment]
Yes, you'r right ! Thank you mate !

File: module/index.js [hunk 1]
```diff
@@ -315,6 +315,7 @@ var ClassicAPI = (function () {
     ClassicAPI.prototype.parseURL = function (unparsedURL) {
         var parsedURL = unparsedURL.replace(/{region}/g, region_e_TO_string(this.region));
         parsedURL = parsedURL.replace(/{endpoint}/g, region_e_TO_endpointString(this.region));
+        console.log(parsedURL);
```
Comment (CoreyTrombley) [inline module/index.js:318]: @ZafixLRP Not sure where this `console.log` came from sorry about that. I did build from the `index.ts` file.

**Comment by lucalaissue** [comment]
Woops

**Comment by lucalaissue** [comment]
I'll remove this tomorow NP

**Pull request merged**